### ISSUE TITLE
ComfyUI crashes when loading `umt5_xxl_fp8_e4m3fn_scaled.safetensors`

### ISSUE THREAD

**Issue by ASnowFlaker** [comfyanonymous/ComfyUI#10471]
### Custom Node Testing

- [x] I have tried disabling custom nodes and the issue persists (see [how to disable custom nodes](https://docs.comfy.org/troubleshooting/custom-node-issues#step-1%3A-test-with-all-custom-nodes-disabled) if you need help)

### Expected Behavior

Using the official wan2.2_ti2v_5B_fp16 workflow
Using ComfyUI 0.3.65 Windows standalone
RTX 3060 VRAM 12 GB, RAM 32 GB

VAE model loads and then CLIP model should load and get to CLIP text encoder node.

### Actual Behavior

CLIP load node highlights when it needs to load the model but after like 7 seconds Comfy UI server crashes. I get "reconnecting" and pressing enter in the server exits cmd.

RAM and VRAM don't go up at all, so there's no OOM involved. 

Same goes for umt5_xxl_fp16.safetensors
Same goes for cpu instead of gpu setting in CLIP node.

I've downloaded, unzipped and run these Comfy UI versions in this order and :
0.3.65 it didn't work
0.3.64 no joy
0.3.60 same fail
0.3.46 (the version that started WAN 2.2 support) it worked!
0.3.50 didn't
0.3.48 worked
0.3.49 worked

So the differences from 0.3.49 to 0.3.50 are:
CIted from github:
    Add Qwen Image model to readme
    Qwen Image model merging node
    qwenLora cannot load properly
    Update template to 0.1.52
    Update frontend to v1.24.4
    Fix RepeatLatentBatch not working on multi dim latents.
    _ui.py import torchaudio safety check
    async API nodes
    Users report gfx1201 is buggy on flux with pytorch attention.
    Not sure if AMD actually support fp16 acc but it doesn't crash.
    Bump pytorch cuda and rocm versions in readme instructions.
    Only show feature flags log when verbose.
    remove creation of non-used asyncio_loop
    Update template & embedded docs
    Support SimpleTuner lycoris lora for Qwen-Image
    fix(Kling Image API Node): do not pass "image_type" when no image
    Update template to 0.1.58
    Update test release package workflow with python 3.13 cu129
    Wan2.2 fun control support.
    Make torchaudio exception catching less specific
    Update template to 0.1.59
    Update release workflow to python3.13 pytorch cu129
    Downgrade frontend for release

Effective changes:
python 3.12.10 -> 3.13.6
pytorch 2.7.1 cu128 -> 2.8.0 cu129
frontend stayed the same: 1.23.4

I've also installed v0.3.65 with python 3.12.10 + torch 2.7.1 cu128. It displays the same fail behavior.

### Steps to Reproduce

Use the official wan2.2_ti2v_5B_fp16 workflow
Use ComfyUI 0.3.65 Windows standalone

### Debug Logs

```powershell
Both 0.3.49 and 0.3.65:

`got prompt
Using pytorch attention in VAE
Using pytorch attention in VAE
VAE load device: cuda:0, offload device: cpu, dtype: torch.bfloat16
Using scaled fp8: fp8 matrix mult: False, scale input: False`

Only 0.3.49 goes on with:

`Requested to load WanTEModel
loaded completely 9.5367431640625e+25 6419.477203369141 True
CLIP/text encoder model load device: cpu, offload device: cpu, current: cpu, dtype: torch.float16`
```

**Comment by jdluzen** [comment]
I am also getting this with `t5-v1_1-xxl-encoder-Q4_K_S.gguf`, 7900XTX.

**Comment by ASnowFlaker** [comment]
I've just tried `t5-v1_1-xxl-encoder-Q8_0.gguf` and for me it works. But I can't use it since it's not `wan` type. But it gave me the idea to try a different tokenizer type for `umt5_xxl_fp8_e4m3fn_scaled` so I've tried `stable_diffusion` it gave me some parameter mismatch. But hey, it didn't crash. What could possibly be different to me compared to the other users? It's not like I'm doing anything differently, it's the vanilla setup. Technically, what's happening is impossible.

**Comment by nomadoor** [comment]
I was also hitting a crash exactly when Load CLIP tried to load umt5_xxl.
In my case, the cause was the sentencepiece version. Since requirements.txt doesn’t pin sentencepiece, a pip update seems to have introduced a library mismatch. 

Running : `pip install --force-reinstall "sentencepiece==0.2.0"`

resolved it for me. Please note this might affect other nodes depending on your environment, so proceed with caution.

**Comment by Kipachan** [comment]
> I was also hitting a crash exactly when Load CLIP tried to load umt5_xxl. In my case, the cause was the sentencepiece version. Since requirements.txt doesn’t pin sentencepiece, a pip update seems to have introduced a library mismatch.
> 
> Running : `pip install --force-reinstall "sentencepiece==0.2.0"`
> 
> resolved it for me. Please note this might affect other nodes depending on your environment, so proceed with caution.

Thanks! It works! I was racking my brain over this crap all day yesterday.
